plugins/exec/vaccel.c

```diff
@@ -7,18 +7,18 @@
 #include <stdio.h>
 #include <vaccel.h>
 
```

Comment:
TODO: maybe we should split this file altogether (on another iteration).

Reply:
Agreed and maybe we should do that for the noop plugin also.